dolphin/task_manager/manager.py

```diff
@@ -67,33 +67,7 @@ def say_hello(self, context, request_spec=None,
             pass
 
     def sync_storage_resource(self, context, storage_id, resource_task):
-        """
-        :param context:
-        :param storage_id:
-        :param resource_task:
-        :return:
-        """
-        try:
-            vol_task = resources.StorageVolumeTask(context, storage_id)
-            pol_task = resources.StoragePoolTask(context, storage_id)
-            [pol_task if resource_task == 'pool_task' else vol_task]\
-                .pop().sync()
-        except Exception as e:
-            LOG.erro(e)
-            raise exception.DolphinException(e)
-
-    def remove_storage_resource(self, context, storage_id, resource_task):
-        """
-        :param context:
-        :param storage_id:
-        :param resource_task:
-        :return:
-        """
-        try:
-            vol_task = resources.StorageVolumeTask(context, storage_id)
-            pol_task = resources.StoragePoolTask(context, storage_id)
-            [pol_task if resource_task == 'pool_task' else vol_task] \
-                .pop().remove()
-        except Exception as e:
-            LOG.erro(e)
-            raise exception.DolphinException(e)
+        LOG.info("Consuming the refresh storage")
```

Comment:
it's more menaningful if storage_id and resource_task are included in the log message.

Reply:
It's added in DB function there, so i don't think, repeated msg required.